astroquery/mpc/core.py

```diff
@@ -1061,12 +1060,20 @@ def _parse_result(self, result, **kwargs):
             # find column headings
             if SKY:
                 # slurp to newline after "h m s"
-                i = text_table.index('\n', text_table.index('h m s')) + 1
+                # raise EmptyResponseError if no ephemeris lines are found in the query response
+                try:
+                    i = text_table.index('\n', text_table.index('h m s')) + 1
+                except ValueError as e:
+                    raise EmptyResponseError(content)
```

Comment:
Not sure that we need/want to dump the full HTML content here. 

Also, is this an EmptyResponseError or maybe a NoResultsWarning? E.g. the user should be able to do something to fix an Error, while a Warning is more like a heads up.

Reply:
Agreed, the response is there so its not empty it just that there are no ephemereis lines to be parsed (table is empty). I will correct that.